scripts/scil_compute_ihMT_maps.py

```diff
@@ -0,0 +1,442 @@
+# -*- coding: utf-8 -*-
+"""
+This script computes four myelin indices maps from the Magnetization Transfer
```

Comment:
Sometimes for complex processing, we add a reference to help users, do you have 1-2 papers describing this acquisition and this method?

(I saw them lower, check scil_recognize_single_bundle.py to see how to include the reference in the argparser)